semantic/src/site/globals/site.overrides

```diff
@@ -162,21 +161,19 @@ a:hover {
   align-items: flex-start;
 }
 
-.spacer-20 { 
-  height: 20px
+.spacer-20 {
+  height: 20px;
 }
 
-.spacer-10 { 
-  height: 10px
+.spacer-10 {
+  height: 10px;
 }
 
 .no-margin-no-padding {
-  padding:0px !important;
-  margin:0px !important;
+  padding: 0px !important;
+  margin: 0px !important;
```

Comment:
Here's that auto-formatting issue again -- keeps getting switched back and forth, one of us has it configured differently.

Reply:
Yes, but I think that's the one we had agreed to use right? With semicolumns and space between keys and values

Reply:
Yeah, it's fine, I'm just wondering why it keeps getting changed away from that and then back. 🤷‍♂️